test/helpers/data/aggregated-data-helper.js

```diff
@@ -287,6 +287,24 @@ module.exports.getAnyExistingTeamId = function () {
   return promise
 }
 
+module.exports.getAnyExistingWorkloadOwnerId = function () {
```

Comment:
Update `getAnyExistingTeamId ` above to match your two (returning id)?

Reply:
Was this added?

Reply:
No....good spot 😄